cfme/utils/appliance/__init__.py

```diff
@@ -922,6 +922,10 @@ def event_listener(self):
             from cfme.utils.events import RestEventListener
             return RestEventListener(self)
 
+    def targeted_refresh_checker(self):
+        from cfme.utils.targeted_refresh import TargetedRefresh
+        return TargetedRefresh()
```

Comment:
**QUESTION**
Why did you add it to the appliance object?

Reply:
I used events as example. I've changed it, is it ok now?